staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -49,6 +49,7 @@ type Config struct {
 
 func NewDefaultConfig(prefix string, copier runtime.ObjectCopier, codec runtime.Codec) *Config {
 	return &Config{
+		Type:   StorageTypeETCD3,
```

Comment:
I'm slightly against this line. We have the defaulting being done already in:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/factory.go#L31

I would prefer to avoid having defaulting in few different places, as if/when we decide to change it again, it will be easier to forget about one of those places.